_apps/auto-assign.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Auto Assign
+description: Add reviewers/assignees to pull requests when pull requests are opened.
+slug: auto-assign
+screenshots:
+- https://github.com/kentaro-m/auto-assign/raw/master/public/assets/screenshots/screenshot.png
+- https://github.com/kentaro-m/auto-assign/raw/master/public/assets/demo.gif
+authors:
+- kentaro-m
+repository: kentaro-m/auto-assign
+host: https://kentaro-m-auto-assign.glitch.me/
+---
+
+A GitHub App built with [Probot](https://github.com/probot/probot) that adds reviewers/assignees to pull requests when pull requests are opened.
+
+## How it works
+* When the pull request is opened, automatically add reviewers/assignees to the pull request.
+* If the number of reviewers is specified, randomly add reviewers/assignees to the pull request. 
```

Comment:
What would you think of adding an option to rotate through the list of reviewers so that everyone is guaranteed to review the same number of pRs?

Reply:
I agree with your idea.
I did not have the point of view that everyone is guaranteed to review the same number of PRs.
As a feature request, I would like to work on it.